src/responses/collection-extra-info.ts

```diff
@@ -5,33 +5,62 @@ import { Metadata } from '../models/metadata';
  * the `collection_details` page type.
  */
 export interface CollectionExtraInfo {
```

Comment:
This diff looks all wonky, but the gist is:

1. Removed the `search_docs_fields` branch and moved all of its sub-entries up a level to match PPS
2. Added JSDoc for all the properties